backend/databasemodel/model.sql

```diff
@@ -1393,6 +1403,7 @@ CREATE TABLE kooste.osm_pisteet (
 	osm_id bigint NOT NULL,
 	osm_type text NOT NULL,
 	geom geometry(POINT, 4326) NOT NULL,
+	tarmo_category text NOT NULL,
```

Comment:
Ah, I see! I misthought that also bus, tram and train stops were coming from OSM.. Do you think it would be okay to set tarmo_category default value to 'Pysäköintipaikka' for osm_pisteet table and 'Pysäköintialue' for osm_alueet table?

Reply:
Yeah it is alright, if we want to set them different.

Also, they may both have the same tarmo category if we desire. Do we want filtering in the UI for these categories?

Reply:
Maybe there is no need to create two separate tarmo categories for this purpose. Let's just set default value for tarmo_category field for osm_pisteet and osm_alueet to 'Pysäköinti'.

And yes, it should be possible to filter according to each possible tarmo_category value.